### PR TITLE
fixes a godawful message

### PR DESCRIPTION
**Pull request by PKPenguin321**
jesus christ who wrote this garbage

### CODE REVIEW

File: code/game/objects/structures/window.dm [hunk 1]
```diff
@@ -320,7 +320,7 @@
 		return
 
 	if(anchored)
-		usr << "<span class='warning'>It is fastened to the floor therefore you can't rotate it!</span>"
+		usr << "<span class='warning'>[src] cannot be rotated while it is fastened to the floor!</span>"
```
Comment (KorPhaeron) [inline code/game/objects/structures/window.dm]: \The [src]

File: code/game/objects/structures/window.dm [hunk 2]
```diff
@@ -340,7 +340,7 @@
...
```
Comment (KorPhaeron) [inline code/game/objects/structures/window.dm]: \The [src]

File: code/game/objects/structures/window.dm (hunk 1 above)
Comment (PKPenguin321) [inline code/game/objects/structures/window.dm]: does [src] not already make it say "The" ?

Comment (Cheridan) [inline code/game/objects/structures/window.dm]: Yeah it's not needed here. Kor's wroooong.

**Comment by optimumtact** [comment]
It was only.lurking
662c0827

File: code/game/objects/structures/window.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/game/objects/structures/window.dm]: Nooooooooooooooo

Comment (KorPhaeron) [inline code/game/objects/structures/window.dm]: I'll delete the evidence

Comment (xxalpha) [inline code/game/objects/structures/window.dm]: You haven't deleted the evidence.

**Pull request merged**